plenty_api/api.py

```diff
@@ -1235,6 +1237,39 @@ def plenty_api_create_booking(self, order_id: int,
                                              data=data)
         return response
 
+
+    def plenty_api_create_property_selection_name(self, property_id: int,
+                                                  lang: str, name: str) -> dict:
+        """
+        Create a name in a specific language for a selection property.
+
+        Parameter:
+            property_id     [int]   -   Plentymarkets property Id which a new
+                                        selection namm will be created for
+            lang            [str]   -   Initial language for selection creation
+            name            [str]   -   Value of the selection name
+
+        Return:
+                            [dict]  -   Return the response object, in case the
+                                        request fails return an error message
+        """
+        if not property_id or not lang or not name:
+            return [{'error': 'missing_parameter'}]
+
+        if utils.get_language(lang=lang) == 'INVALID_LANGUAGE':
+            return {'error': 'invalid_language'}
+
+        json = {'propertyId': property_id, 'lang': lang, 'name': name,}
+
+        if not utils.sanity_check_json(route_name='property_selection',
+                                       json=json):
+            return {'error': 'invalid_json'}
```

Comment:
I think this sanity check isn't required, I mean you built the JSON so you can trust it as well.